setup.py

```diff
@@ -42,6 +42,7 @@ def find_version(*file_paths):
 
 
 install_requires = [
+    'six>=1.12',
```

Comment:
Isn't that required only for Python2 & Python3 compatibility? I thought we wouldn't need that for Cylc, or is it due to one of our dependencies?

Reply:
For some reason Travis is installing 1.11, yet graphene indirectly requires 1.12  ... Works in my virtual env (`pip install -e .["all"]`)

Reply:
It worked, but I'll try to hunt down the other packages installing `six`

Reply:
Graphene has:
```
    install_requires=[
        "six>=1.10.0,<2",
        "graphql-core>=2.1,<3",
        "graphql-relay>=2,<3",
        "aniso8601>=3,<=7",
    ],
```
and `graphql-relay` has:
```
    install_requires=[
        'six>=1.12',
        'graphql-core>=2.2,<3',
        'promise>=2.2,<3'
    ],
```
Which is compatible, yet somehow Travis installs `1.11` .... pip version? works by hand, and in Travis if I explicitly set `six>=1.12`..

Reply:
If we need to bodge Travis we could put `pip install six>=1.12` in `.travis/install`.

Reply:
Good idea :point_up:

Reply:
Done.